.github/workflows/diff-change-to-dist.yaml

```
@@ -51,9 +52,4 @@ jobs:
               diff +
               '\n```'
 
-            github.rest.issues.createComment({
-              issue_number: context.payload.pull_request.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: commentText
-            })
+            await commentOnPr({github,context}, 'Diff release', commentText)
```

Love how this is all tucked away and tidy now 😊 

Worth running these snippets through ESLint?

```suggestion
            await commentOnPr({ github, context }, 'Diff release', commentText)
```